plans/init.pp

```diff
@@ -165,6 +173,9 @@
     $reboot_message_group = pick($reboot_message,
                                   $group_vars['patching_reboot_message'],
                                   'NOTICE: This system is currently being updated.')
+    $update_provider_group = pick($update_provider,
+                              $group_vars['update_provider'],
```

Comment:
```suggestion
                              $group_vars['patching_update_provider'],
```

By convention i've been trying to name my variables `patching_xxx` so this suggestion just simply renames to that convention. :)